src/frontend/sass/splash-screen.scss

```diff
@@ -0,0 +1,44 @@
+@import './mat-colors';
+// Custom theme support
+@import './custom';
+app-root {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  color: $white;
+}
+
+body {
+  background-image: url(/assets/login-bg.jpg);
+  margin: 0;
+  padding: 0;
+}
+
+.splash-screen-loader {
+  border: 16px solid $white-87-opacity;
+  border-radius: 50%;
+  border-top: 16px solid map-get($mat-blue, 500);
```

Comment:
Could we style the spinner the same and the material design one? Here is an example in pure css https://codepen.io/jczimm/pen/vEBpoL and here is an example of one we use in the application https://material.angular.io/components/progress-spinner/overview. A bit of modification to the codepen (like removing the colour changes) could make it fit quite well.